llvm/include/llvm/MC/MCObjectStreamer.h

```diff
@@ -141,8 +141,8 @@ class MCObjectStreamer : public MCStreamer {
                          SMLoc Loc) override;
   void emitDwarfLocDirective(unsigned FileNo, unsigned Line, unsigned Column,
                              unsigned Flags, unsigned Isa,
-                             unsigned Discriminator,
-                             StringRef FileName) override;
+                             unsigned Discriminator, StringRef FileName,
+                             StringRef Location = {}) override;
```

Comment:
Nit: `Location` is a bit misleading here. AFAICT, this will print anything after the `.loc` directive.

`Comment` or `Note` might describe the argument purpose better, even if we do happen to use it for whatever `DebugLoc.print()` generates.